app/views/devise/registrations/new.html.erb

```diff
@@ -0,0 +1,36 @@
+<%= render partial:'/shared/nav_page', locals: {title: 'Sign Up', backurl:
+  users_path} %>
+  <section class="sign-section">
+    <div class="form-registration">
+      <%= form_for(resource, as: resource_name, url: registration_path(resource_name)) do |f| %>
+      <%= render "devise/shared/error_messages", resource: resource %>
+    
+      <div class="custom-form-field">
+        <%= f.text_field :name, autofocus: true, placeholder:'Full Name',
+        class:'custom-form-field-text', required:true %>
+      </div>
+    
+      <div class="custom-form-field">
+        <%= f.email_field :email, autofocus: true, required:true,
+        placeholder:'E-mail',class:'custom-form-field-text' %>
+      </div>
+    
+      <div class="custom-form-field">
+        <%= f.password_field :password, autocomplete: "new-password",
+        placeholder:'Password (min 6 characters)',class:'custom-form-field-text' %>
+      </div>
```

Comment:
- [ ] Kindly ensure that the forms look like the given behance design. In this case, let's make sure that the background of each of the form inputs are identical to the original design
- [ ] You are using a font-family different from what was specified in the original design. Please let us stick as closely as possible to the design guidelines 👍 
- [ ] Please apply these changes to all the forms in your application
### Design Guideline
![image](https://github.com/roniy68/budget-app/assets/46839548/4cb5bf08-8f65-4002-9d13-5af791a78a1a)
### Your Implementation
![image](https://github.com/roniy68/budget-app/assets/46839548/65dd3986-1a98-4233-85e9-cda92ef2b89e)

Reply:
🔥 Solved